sdk/textanalytics/azure-ai-textanalytics/tests/test_analyze.py

```diff
@@ -47,6 +47,17 @@
 # pre-apply the client_cls positional argument so it needn't be explicitly passed below
 TextAnalyticsClientPreparer = functools.partial(_TextAnalyticsClientPreparer, TextAnalyticsClient)
 
+TextAnalyticsCustomPreparer = functools.partial(
+    TextAnalyticsPreparer,
+    textanalytics_custom_text_endpoint="https://fakeendpoint.cognitiveservices.azure.com",
```

Comment:
why are these variables moved from the testcase to the test files?

Reply:
If I'm not mistaken, they needed to be separated since these env vars don't exist in the UsGov/China cloud.

Reply:
@kristapratico @iscai-msft  If I don't remove these variables, I get an error when rerun textanalytics weekly pipeline. 
Error message: `azure_devtools.scenario_tests.preparers:preparers.py:82 Preparer failure when creating resource PowerShellPreparer for test <test_auth.TestAuth object at 0x111e26190>: 'TEXTANALYTICS_CUSTOM_TEXT_ENDPOINT'`
For more details please check [here](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1283946&view=logs&j=a7bc4bf7-faea-50a8-6676-78ef3b853b40&t=fc991221-8f54-5a2a-9702-94d71b289a0e).

For example `test_auth.py::TestAuth::test_active_directory_auth`, I have debugged this test locally. When the code is executed to [devtools_testutils/powershell_preparer.py](https://github.com/Azure/azure-sdk-for-python/blob/main/tools/azure-sdk-tools/devtools_testutils/powershell_preparer.py#L65) , it will get the corresponding values of these variables ( in TextAnalyticsPreparer) from the environment variables one by one, if not, an exception will be thrown. Because the values of these variables are not passed in UsGov/China cloud, the test will fail before the testcase actually runs.

So I moved variables from the testcase to the test files. Or do you have better suggestions?